osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -23,16 +25,20 @@ public class OsuSliderBar<T> : SliderBar<T>, IHasTooltip where T : struct
         private readonly Box leftBox;
         private readonly Box rightBox;
 
+        public Func<T, string> TooltipTextFunc { get; set; }

```

Comment:
why not just make `TooltipText` `virtual` and make classes for special cases? i think that's a lot more elegant than this.